src/snowcli/zipper.py

```diff
@@ -47,7 +47,7 @@ def zip_dir(source: Path, dest_zip: Path) -> None:
     with ZipFile(dest_zip, "w", ZIP_DEFLATED, allowZip64=True) as package_zip:
         for file in files_to_pack:
             log.debug("Adding %s to %s", file, dest_zip)
-            package_zip.write(file, arcname=file.relative_to(source.absolute()))
+            package_zip.write(file, arcname=_get_arcname(file, source))
```

Comment:
How about separating this in two two steps:
1. Include users code
2. Add depndencies / packages

In this way we can treat `.packages` as ignored directory. And have and explicit step that will add their content. This may help in further refactoring. WDYT?

Reply:
Done.  Now the function is called twice from snowpark_package() - once for source code, second time, for packages